recipes/catimg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: catimg
+  version: "1.1.1"
+
+source:
+  fn: catimg-1.1.1.tar.gz
+  url: https://pypi.python.org/packages/source/c/catimg/catimg-1.1.1.tar.gz
+  md5: 94227c65936a5e4cada9260efe0125c4
+
+build:
+  entry_points:
+    - catimg = catimg.__main__:main
+  number: 0
```

Comment:
Done below.

FWIW I did included 2 spaces in that code snippet, but it makes them appear as 1. 😒